tests/bundle-test.js

```diff
@@ -87,17 +90,17 @@ async function runAll() {
   const didCreationDetails = {
     authenticationKey: {
       publicKey: authPublicKey,
-      type: 'Ed25519',
+      type: 'ed25519',
     },
     encryptionKey: {
       publicKey: encPublicKey,
-      type: 'X25519',
+      type: 'x25519',
     },
   }
   const testDid = Did.LightDidDetails.fromDetails(didCreationDetails)
   if (
-    testDid.did !==
-    `did:kilt:light:01${address}:z1Ac9CMtYCTRWjetJfJqJoV7FcPDD9nHPHDHry7t3KZmvYe1HQP1tgnBuoG3enuGaowpF8V88sCxytD7HjY6Doe`
+    testDid.uri !==
+    `did:kilt:light:01${address}:z1Ac9CMtYCTRWjetJfJqJoV7FcPDD9nHPHDHry7t3KZmvYe1HQP1tgnBuoG3enuGaowpF8V88sCxytDPDy6ZxhW`
```

Comment:
This looks like a breaking change to me. The light did encoded stuff should not change after the last commits on develop, no? @ntn-x2 ?

Reply:
After research it turns out, that this version is the correct and non-breaking one